app/admin/projects.rb

```diff
@@ -0,0 +1,3 @@
+ActiveAdmin.register Project do
+  permit_params :name
```

Comment:
If there is no form, then why are we permitting a param? If this is not finalized or finished, then delete this file for now, you can add this later

Reply:
This is built in, I have a page of Projects at admin side of this.